src/views/blocks/parts/details.php

```diff
@@ -130,12 +141,11 @@
 		<?php
 		// Event Cost
 		if ( ! empty( $cost ) ) : ?>
-
-            <dt> <?php esc_html_e( 'Cost:', 'the-events-calendar' ) ?> </dt>
-            <dd class="tribe-events-event-cost"> <?php esc_html_e( $cost ); ?> </dd>
+			<dt> <?php esc_html_e( 'Cost:', 'the-events-calendar' ); ?> </dt>
+			<dd class="tribe-events-event-cost"> <?php echo esc_html( $cost ); ?> </dd>
```

Comment:
:no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: 6696a3c83d46016669e910bc03396894</sub>

Reply:
I think the bot is tripping on the space _inside_ the <dd>?